README.md

```diff
@@ -335,175 +350,43 @@ conda create --name nemo python==3.10.12
 conda activate nemo
 ```
 
-Install PyTorch using their
-[configurator](https://pytorch.org/get-started/locally/):
+#### Pick the right version
+
+NeMo-Framework publishes pre-built wheels with each release.
+To install nemo_toolkit from such a wheel, use the following installation method:
 
 ```bash
-conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia
+pip install "nemo_toolkit"
```

Comment:
Does this work? Don't we need "nemo_toolkit[all]" or a specific domain? Also can we switch to "nemo_framework" ?